.github/scripts/dapr_bot.js

```diff
@@ -26,7 +26,8 @@ const owners = [
     'tanvigour',
     'yaron2',
     'rabollin',
-    'ashiquemd'
+    'ashiquemd',
+    'mikeee'
```

Comment:
Please put names in alphabetical (I know yours isn’t the first to be out of sort!)